### DISCUSSION TITLE
Also include in the FAQ that the user can check the hosted App `renovateVersion` via logs at `developer.mend.io`

### DISCUSSION THREAD

**Discussion by blakeNaccarato** [renovatebot/renovate#28662] [Suggest an Idea]
### Tell us more.

Discussion https://github.com/renovatebot/renovate/discussions/22738 explains that the hosted app `renovateVersion` lags the GitHub releases somewhat, and the [related FAQ entry](https://docs.renovatebot.com/faq/#when-is-the-mend-renovate-app-updated-with-new-renovate-versions) (and PR: https://github.com/renovatebot/renovate/pull/22816) explains this delay.

This FAQ entry is certainly helpful, but it does raise the question, "Well, what version is the hosted app actually running on, then?" Could this FAQ entry be updated (or link to an additional entry with the question I just posed) with an "actionable" step recommending a user to visit https://developer.mend.io/ and check the logs to see what their `renovateVersion` is? I stumbled on the logs when looking around for an answer to this question, and it might be helpful to say as much in the FAQ. Thanks!

**Comment by HonkingGoose** [comment]
Good idea! I'm adding new instructions here:

- #28704

Feel free to check my instructions! I'm curious if every user needs the _same_ steps. I'm in multiple organizations, so I'm seeing a username select screen first. But maybe you don't see that screen at all, if you're only using Renovate as a "simple one user account".
